### ISSUE TITLE
Svelte language server initialization fails

### ISSUE THREAD

**Issue by heiwiper** [emacs-lsp/lsp-mode#3726]
### Thank you for the bug report

- [X] I am using the latest version of `lsp-mode` related packages.
- [X] I checked [FAQ](https://emacs-lsp.github.io/lsp-mode/page/faq/) and [Troubleshooting](https://emacs-lsp.github.io/lsp-mode/page/troubleshooting/) sections
- [X] You may also try reproduce the issue using clean environment using the following command: `M-x lsp-start-plain`

### Bug description

Svelte language server doesn't start correctly, it is stuck in `starting` state and it outputs the following messages on the buffer `*Messages*`:
```
LSP :: Connected to [svelte-ls:25682/starting].
LSP :: Error from the Language Server: Request initialize failed with message: Cannot read properties of undefined (reading 'debug') (Internal Error)
```

### Steps to reproduce

1. Install the `svelte-ls` language server.
2. Assuming `npm` is installed on the machine, create a template project `npm create vite@latest myapp -- --template svelte`
3. Open the file `./myapp/src/App.svelte`
4. Start the `lsp` server.
5. The echo area should print that the server is starting and then an LSP error message should be printed afterwards.

### Expected behavior

The Svelte language server's state should be set to `started` and the buffer should get connect to the language server.

### Which Language Server did you use?

svelte-ls

### OS

Linux

### Anything else?

When trying to start the `lsp` server another time after the error is printed, the buffer would keep trying to connect continuously.

**Comment by jrbrodie77** [comment]
I just did a fresh re-install of Doom + Emacs 28 and I'm having the same problem.

**Comment by blame2020** [comment]
This looks like an upstream bug.
https://github.com/sveltejs/language-tools/issues/1645

**Comment by predragnikolic** [comment]
This looks like it is fixed with `svelte-language-server` v0.14.36
